connector/data-source-handler.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+class DataSourceHandler {
+  static createDataSource(workspace, id, data, cb) {
+    function create(next) {
+      workspace.addDataSource(id, data, function(err) {
+        next(err);
```

Comment:
This is the line that is referencing nothing. Need to clean up the code to make your intents more clear.